### PR TITLE
[need-review] Add erct

### PR DESCRIPTION
**Pull request by hiepnm**

### CODE REVIEW

File: spec/lib/coin-address-validators_spec.rb [hunk 1]
```diff
@@ -69,7 +76,7 @@
     describe "#parse_cash_addr" do
       context "invalid address" do
         it "handles gracefully" do
-          legacy_address, ok = CoinAddressValidators["bch"].parse_cash_addr("ABC")
+          legacy_address, ok = CoinAddressValidators::BchValidator.new.parse_cash_addr("ABC")
```
Comment (dunghuynh) [inline spec/lib/coin-address-validators_spec.rb]: Why we need to change this?

Comment (hiepnm) [reply spec/lib/coin-address-validators_spec.rb]: If not, it's failed in this line.
CoinAddressValidators["bch"] only return BchValidator when it call `valid?` method. (only `valid?` method reach to `inner_validator_class`)

Comment (dunghuynh) [reply spec/lib/coin-address-validators_spec.rb]: I means we didn't change anything related to this code, why does it failed?

Comment (dunghuynh) [reply spec/lib/coin-address-validators_spec.rb]: Oh i see, you updated the validator. Got it

Comment (hiepnm) [reply spec/lib/coin-address-validators_spec.rb]: It's failed on master branch now. So, this line fixed it.

Comment (dunghuynh) [reply spec/lib/coin-address-validators_spec.rb]:
```
def parse_cash_addr(address)
  raise "unimpleted" unless inner_validator.has_method?(:parse_cash_addr)
  inner_validator.parse_cash_addr(address) 
end
```

Comment (hiepnm) [reply spec/lib/coin-address-validators_spec.rb]: Fixed!

**Comment by dunghuynh** [comment]
Published 0.0.7 to rubygems

**Pull request merged**